src/index.js

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable no-unused-vars */
+import _ from 'lodash'; // Unused vars disabled for lodash
+import './style.css';
+import {
+  changeCompletedState, dataBase, loadValues, updateValues,
+} from './functions.js';
+import { addTask, removeAll, removeTask } from './add-remove.js';
+import Task from './taskClass.js';
+
+export const completedTasks = [];
+let currentSelected = null;
+function renderTask(task) {
```

Comment:
I believe it would be great to use the arrow notation for your functions to follow [KISS best practices](https://medium.com/infancyit/kiss-keep-it-simple-smart-javascript-habits-b019ce14b5ec).